src/main/java/seedu/address/logic/commands/UntagCommand.java

```diff
@@ -0,0 +1,118 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Birthday;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
+public class UntagCommand extends EditCommand {
+
+    public static final String COMMAND_WORD = "untag";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Removes a tag from the "
+            + "details of the person identified "
+            + "by the index number used in the displayed person list. "
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_TAG + "friend";
+
+    public static final String MESSAGE_REMOVE_PERSON_SUCCESS = "Removed tag from Person: %1$s";
```

Comment:
Is it possible for this description to be more specific? Eg. Removed tag from Adam instead?

Reply:
Yep will update in the next commit